ddtrace/ext/__init__.py

```diff
@@ -22,3 +22,17 @@ class SpanKind(object):
     SERVER = "server"
     PRODUCER = "producer"
     CONSUMER = "consumer"
+
+
+EXIT_SPAN_TYPES = frozenset(
+    {
+        SpanTypes.CACHE,
+        SpanTypes.CASSANDRA,
+        SpanTypes.ELASTICSEARCH,
+        SpanTypes.GRPC,
+        SpanTypes.HTTP,
+        SpanTypes.REDIS,
+        SpanTypes.SQL,
+        SpanTypes.WORKER,
+    }
+)
```

Comment:
I think I just lack context here, but why are these specifically `EXIT_SPAN_TYPES`? I see they're used in line 200 on `SpanCodeOriginProcessor.on_span_start()` to check the started span type, and if it's not one of these, we return early. Are these just the exit span types we're collecting the code origin data for currently, and it will expand later?

Reply:
Here we want to identify any span that is representing a call to an external service. AFAIK these are the only such span types that we currently support. This set could grow in the future if we introduce new types